src/website/app/demos/Card/examples/card/orderOfSections.js

```diff
@@ -38,38 +41,37 @@ export default {
   title: 'Order of sections',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description: 'The sections of Card are designed to work in any order.',
+  description: `There is no 'one true way' to lay out a Card — they are flexible enough to accept different component arrangements.`,
   scope: { Card, CardBlock, CardImage, CardTitle, DemoLayout },
   source: `
     <DemoLayout>
       <Card>
-        <CardImage src="http://fillmurray.com/1000/563" alt="Bill Murray" />
-        <CardTitle minor>Card title</CardTitle>
+        <CardImage src="/images/stibnite.jpg" alt="Stibnite" />
+        <CardTitle minor>Stibnite</CardTitle>
         <CardBlock>
-          Light years star stuff harvesting star light citizens of distant
-          epochs encyclopaedia galactica vastness is bearable only through love,
-          shores of the cosmic ocean!
+          Stibnite, sometimes called antimonite, is a sulfide mineral with the
+          formula Sb₂S₃. This soft grey material crystallizes in an orthorhombic
+          space group. It is the most important source for the metalloid
+          antimony.
         </CardBlock>
       </Card>
 
       <Card>
-        <CardTitle>Card title</CardTitle>
-        <CardImage src="http://fillmurray.com/1001/563" alt="Bill Murray" />
+        <CardTitle>Lazurite</CardTitle>
+        <CardImage src="/images/lazurite.jpg" alt="Lazurite" />
         <CardBlock>
-          Light years star stuff harvesting star light citizens of distant
-          epochs encyclopaedia galactica vastness is bearable only through love,
-          shores of the cosmic ocean!
+          Lazurite crystallizes in the isometric system although well formed
+          crystals are rare. It is usually massive and forms the bulk of the
+          gemstone lapis lazuli.
         </CardBlock>
       </Card>
 
       <Card>
-        <CardTitle>Card title</CardTitle>
+        <CardTitle>Manganite</CardTitle>
         <CardBlock>
-          Light years star stuff harvesting star light citizens of distant
-          epochs encyclopaedia galactica vastness is bearable only through love,
-          shores of the cosmic ocean!
+          Crystals of manganite are prismatic and deeply striated parallel to their length; they are often grouped together in bundles. The color is dark steel-grey to iron-black, and the luster brilliant and submetallic.
         </CardBlock>
-        <CardImage src="http://fillmurray.com/1002/563" alt="Bill Murray" />
+        <CardImage src="/images/manganite.jpg" alt="Olivine" />
```

Comment:
Where do all of these images come from?  Are they royalty-free?  Do we need to provide credit?